presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -16,8 +16,11 @@
 import io.airlift.configuration.Config;
 import io.airlift.configuration.LegacyConfig;
 
+import javax.validation.constraints.NotNull;
+
 public class FeaturesConfig
 {
+    public static final String DEFAULT_RESOURCE_GROUP_MANAGER = "default";
```

Comment:
Maybe we can come up with a better name than `default`.. something that describes the rules system